sdk/template/Azure.Template/tests/Samples/Sample1_HelloWorld.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.Identity;
+#region Snippet:Azure_Template_Namespaces
+using Azure.Template.Models;
+#endregion
+using NUnit.Framework;
+
+namespace Azure.Template.Tests.Samples
+{
+    public partial class Sample1_HelloWorld: SamplesBase<TemplateClientTestEnvironment>
+    {
+        [Test]
+        [SyncOnly]
+        public void GettingASecret()
+        {
+            var endpoint = TestEnvironment.KeyVaultUri;
```

Comment:
We try to avoid using `var` throughout our source, especially for samples. A few instances throughout this PR to also update.

Reply:
updated